test/jsx.tsx

```diff
@@ -36,6 +36,16 @@ describe('snabbdom', function() {
       }));
     });
 
+    it('supports sel property shorthand', function() {
+      const vnode = <div sel="#id.foo.bar"></div>;
+
+      assert.equal(JSON.stringify(vnode), JSON.stringify({
+        sel: 'div#id.foo.bar',
+        data: {},
+        children: [],
+      }));
+    });
+
```

Comment:
I don't take it for granted that this should be supported. I'm thinking that since snabbdom JSX is so different than React JSX (there is no `<h href="foo">foo</h>`, you have to do `<h attrs={{ href: 'foo' }}/>foo</h>`), we have the freedom to let this `h` feature reach JSX-land. snabbdom JSX attributes are sharing HTML attribute namespace. I rather like that. It saves us a lot of trouble React code is dealing with due to this. OK. I'm OK with this.

Reply:
I'm not sure what to make of this comment. The reasoning behind having sel was developer UX. 

It's much easier and readable to write

```jsx
<div sel=".profile">My Name</div>
```

and equivalent css has

```
.profile {
 // some css props
}
```

makes it easier to just grep for css selectors and see both style/dom tree come up in search.

rather than

```
<div class={{profile: true}}>My Name</div>
// or 
<div attrs={{class: `profile`}}>My Name</div>
```

Plus `h('tag#id.class')` is optimized because snabbdom only does id class attribute adding at tag create time and doesn't diff it after, so its a minor perf win.

Re: snabbdom namespacing, I really love it. It's better than react because react can only do props. With snabbdom you can have custom modules for animations, hooks, datasets, listen to custom events e.t.c

Reply:
I apologize. I meant to write "snabbdom JSX attributes are *not* sharing HTML attribute namespace". I'm alright with this change. Your reasoning and examples are great.